Hikari error occurred when i using scaling.

Two resource:
```
(Old): database_team_default
(New): database_team_new
```


Old TABLE RULE:
```
CREATE SHARDING TABLE RULE email_campaign (
    DATANODES("database_team_default.EMAIL_CAMPAIGN")
);
```

Create SCALING RULE:
```
CREATE SHARDING SCALING RULE sharding_scaling(
    INPUT(WORKER_THREAD=40, BATCH_SIZE=1000),
    OUTPUT(WORKER_THREAD=40, BATCH_SIZE=1000),
    STREAM_CHANNEL(TYPE(NAME=MEMORY, PROPERTIES("block-queue-size"=10000))),
    COMPLETION_DETECTOR(TYPE(NAME=IDLE, PROPERTIES("incremental-task-idle-minute-threshold"=30))),
    DATA_CONSISTENCY_CHECKER(TYPE(NAME=DATA_MATCH, PROPERTIES("chunk-size"=1000)))
);
```

New Table Rule:
```
ALTER SHARDING TABLE RULE email_campaign (
    DATANODES("database_team_new.EMAIL_CAMPAIGN_TEAM_${['0','1','2']}"),
    TABLE_STRATEGY(
        TYPE=standard,
        SHARDING_COLUMN=email_campaign_team_id,
        SHARDING_ALGORITHM=email_campaign_table_sharding_strategy
    ),
    KEY_GENERATE_STRATEGY(
        COLUMN=email_campaign_id,
        TYPE(NAME=snowflake)
    )
);
```

Then Automatic scaling triggered...

And then:
<img width="1669" alt="image" src="https://user-images.githubusercontent.com/22064617/165017883-acc4624e-ed7e-4353-a4e9-a96e100fbbbb.png">

```
[ERROR] 2022-04-25 03:30:29.464 [ShardingSphere-Scaling-execute-0] o.a.s.d.p.c.i.AbstractImporter - flush failed 0/3 times.
java.sql.SQLException: HikariDataSource HikariDataSource (HikariPool-49) has been closed.
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:96)
	at org.apache.shardingsphere.driver.jdbc.core.connection.ConnectionManager.createConnection(ConnectionManager.java:369)
	at org.apache.shardingsphere.driver.jdbc.core.connection.ConnectionManager.createConnections(ConnectionManager.java:338)
	at org.apache.shardingsphere.driver.jdbc.core.connection.ConnectionManager.getConnections(ConnectionManager.java:327)
	at org.apache.shardingsphere.infra.executor.sql.prepare.driver.DriverExecutionPrepareEngine.group(DriverExecutionPrepareEngine.java:89)
	at org.apache.shardingsphere.infra.executor.sql.prepare.AbstractExecutionPrepareEngine.prepare(AbstractExecutionPrepareEngine.java:68)
	at org.apache.shardingsphere.driver.jdbc.core.statement.ShardingSpherePreparedStatement.initBatchPreparedStatementExecutor(ShardingSpherePreparedStatement.java:576)
	at org.apache.shardingsphere.driver.jdbc.core.statement.ShardingSpherePreparedStatement.executeBatch(ShardingSpherePreparedStatement.java:557)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.executeBatchInsert(AbstractImporter.java:172)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.doFlush(AbstractImporter.java:147)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.tryFlush(AbstractImporter.java:132)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.flushInternal(AbstractImporter.java:123)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.lambda$flush$2(AbstractImporter.java:114)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1508)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.flush(AbstractImporter.java:112)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.write(AbstractImporter.java:93)
	at org.apache.shardingsphere.data.pipeline.core.importer.AbstractImporter.doStart(AbstractImporter.java:79)
	at org.apache.shardingsphere.data.pipeline.api.executor.AbstractLifecycleExecutor.start(AbstractLifecycleExecutor.java:41)
	at org.apache.shardingsphere.data.pipeline.api.executor.AbstractLifecycleExecutor.run(AbstractLifecycleExecutor.java:61)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)
[INFO ] 2022-04-25 03:30:30.465 [ShardingSphere-Scaling-execute-0] o.a.s.d.p.c.i.AbstractImporter - importer write done, rowCount=28000, finishedByBreak=false
[INFO ] 2022-04-25 03:30:30.465 [ShardingSphere-Scaling-execute-1] o.a.s.d.p.core.task.InventoryTask - importer onSuccess, taskId=database_team_default.EMAIL_CAMPAIGN#0
```

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedBigintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedBigintHandler.java
@@ -31,6 +31,9 @@ public final class MySQLUnsignedBigintHandler implements MySQLDataTypeHandler {
     
     @Override
     public Serializable handle(final Serializable value) {
+        if (value == null) {
+            return null;
+        }
         long longValue = (long) value;
         return 0 > longValue ? BIGINT_MODULO.add(BigInteger.valueOf(longValue)) : longValue;
     }

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedBigintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedBigintHandler.java
@@ -31,7 +31,7 @@ public final class MySQLUnsignedBigintHandler implements MySQLDataTypeHandler {
     
     @Override
     public Serializable handle(final Serializable value) {
-        if (value == null) {
+        if (null == value) {
             return null;
         }
         long longValue = (long) value;

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedIntHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedIntHandler.java
@@ -30,7 +30,7 @@ public final class MySQLUnsignedIntHandler implements MySQLDataTypeHandler {
     
     @Override
     public Serializable handle(final Serializable value) {
-        if (value == null) {
+        if (null == value) {
             return null;
         }
         int intValue = (int) value;

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedIntHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedIntHandler.java
@@ -30,6 +30,9 @@ public final class MySQLUnsignedIntHandler implements MySQLDataTypeHandler {
     
     @Override
     public Serializable handle(final Serializable value) {
+        if (value == null) {
+            return null;
+        }
         int intValue = (int) value;
         return intValue < 0 ? INT_MODULO + intValue : intValue;
     }

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedMediumintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedMediumintHandler.java
@@ -30,7 +30,7 @@ public final class MySQLUnsignedMediumintHandler implements MySQLDataTypeHandler
     
     @Override
     public Serializable handle(final Serializable value) {
-        if (value == null) {
+        if (null == value) {
             return null;
         }
         int intValue = (int) value;

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedMediumintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedMediumintHandler.java
@@ -30,6 +30,9 @@ public final class MySQLUnsignedMediumintHandler implements MySQLDataTypeHandler
     
     @Override
     public Serializable handle(final Serializable value) {
+        if (value == null) {
+            return null;
+        }
         int intValue = (int) value;
         return 0 > intValue ? MEDIUMINT_MODULO + intValue : intValue;
     }

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedSmallintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedSmallintHandler.java
@@ -30,7 +30,7 @@ public final class MySQLUnsignedSmallintHandler implements MySQLDataTypeHandler 
     
     @Override
     public Serializable handle(final Serializable value) {
-        if (value == null) {
+        if (null == value) {
             return null;
         }
         short shortValue = (short) value;

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedSmallintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedSmallintHandler.java
@@ -30,6 +30,9 @@ public final class MySQLUnsignedSmallintHandler implements MySQLDataTypeHandler 
     
     @Override
     public Serializable handle(final Serializable value) {
+        if (value == null) {
+            return null;
+        }
         short shortValue = (short) value;
         return 0 > shortValue ? SMALLINT_MODULO + shortValue : shortValue;
     }

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedTinyintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedTinyintHandler.java
@@ -30,7 +30,7 @@ public final class MySQLUnsignedTinyintHandler implements MySQLDataTypeHandler {
     
     @Override
     public Serializable handle(final Serializable value) {
-        if (value == null) {
+        if (null == value) {
             return null;
         }
         byte byteValue = (byte) value;

--- a/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedTinyintHandler.java
+++ b/shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/column/value/impl/MySQLUnsignedTinyintHandler.java
@@ -30,6 +30,9 @@ public final class MySQLUnsignedTinyintHandler implements MySQLDataTypeHandler {
     
     @Override
     public Serializable handle(final Serializable value) {
+        if (value == null) {
+            return null;
+        }
         byte byteValue = (byte) value;
         return 0 > byteValue ? TINYINT_MODULO + byteValue : byteValue;
     }

